mlir/test/Dialect/Arith/mesh-spmdize.cpp

```diff
@@ -0,0 +1,17 @@
+// RUN: mlir-opt \
+// RUN:   --pass-pipeline="builtin.module(func.func(mesh-spmdization))" \
```

Comment:
Is there no public option for this pass?

Reply:
Yes it does, but it is defined as `InterfacePass<"mesh-spmdization", "mlir::FunctionOpInterface">` which apparently requires the above. We can look into that in a separate PR.